ckanext/unhcr/actions.py

```diff
@@ -1253,20 +1253,23 @@ def user_create(up_func, context, data_dict):
 
 @toolkit.chained_action
 def user_update(up_func, context, data_dict):
-    user_id = toolkit.get_or_bust(data_dict, 'id')
-    m = context.get('model', model)
-    user_obj = m.User.get(user_id)
 
-    if user_obj is not None and is_saml2_user(user_obj):
+    if not context.get('ignore_auth', False):
```

Comment:
The default return vale for `.get()` is None
```suggestion
    if not context.get('ignore_auth'):
```